Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/D3D12Quad.cpp

```diff
@@ -0,0 +1,832 @@
+#include "pch.h"
+#include "stdafx.h"
+#include "D3D12Quad.h"
+
+inline std::string HrToString(HRESULT hr)
+{
+    char s_str[64] = {};
+    sprintf_s(s_str, "HRESULT of 0x%08X", static_cast<UINT>(hr));
+    return std::string(s_str);
+}
+
+class HrException : public std::runtime_error
+{
+public:
+    HrException(HRESULT hr) : std::runtime_error(HrToString(hr)), m_hr(hr) {}
+    HRESULT Error() const { return m_hr; }
+private:
+    const HRESULT m_hr;
+};
+
+D3D12Quad::D3D12Quad(UINT width, UINT height, std::wstring name) :
+    m_width(width),
+    m_height(height),
+    m_title(name),
+    m_frameIndex(0),
+    m_viewport(0.0f, 0.0f, static_cast<float>(width), static_cast<float>(height)),
+    m_scissorRect(0, 0, static_cast<LONG>(width), static_cast<LONG>(height)),
+    m_rtvDescriptorSize(0)
+{
+    m_aspectRatio = static_cast<float>(width) / static_cast<float>(height);
+}
+
+void D3D12Quad::OnInit()
+{
+    auto title = GetTitle();
+    fileNames = {
+        Win32Application::GetAssetPath(L"dog.jpg").c_str(),
+        Win32Application::GetAssetPath(L"cat.jpg").c_str(),
+        Win32Application::GetAssetPath(L"hummingbird.jpg").c_str()
+    };
+    fileIndex = 0;
+    updateCounter = 0;
+    LoadPipeline();
+    LoadAssets();
+    is_initialized = true;
```

Comment:
[nit] Win32 apps are predominantly camel case for locals and member fields. Even WinRT C++ apps use camel case for locals (e.g. https://github.com/microsoft/windows-appsample-photo-editor/blob/master/PhotoEditor/Photo.cpp) along with Win32, DirectX, and pretty much every other Microsoft codebase, with the rare oddball exception of ONNX Runtime (probably because the founders originated from TensorFlow) and certain classes in WinRT C++ projections (because those mimic the STL). Granted, this is a demo on using ORT, but the app itself is win32 (and it's good to be internally self-consistent with all the other variables like `updateCounter` and `fileIndex`).